requirements.txt

```diff
@@ -1,2 +1,3 @@
 pre-commit==2.19.0
 gitlint==0.19.1
+pytest==8.0.0
```

Comment:
I don't think pytest should be added as a dependency for the whole app (we use this top-level requirements file for pre-commit).
pytest is already installed as a dev dependency in `/server/pyproject.toml`